yt/yt/tests/integration/queues/test_queue_agent.py

```diff
@@ -3235,6 +3146,142 @@ def test_multicell_export(self, queue_external_cell_tag):
+
+        export_dir = "//tmp/export"
+        self._create_export_destination(export_dir, queue_id)
+
+        export_period_seconds = 5
+        set("//tmp/q/@static_export_config", {
+            "default": {
+                "export_directory": export_dir,
+                "export_period": export_period_seconds * 1000,
+            }
+        })
+
+        tx = start_transaction()
+        lock(export_dir, mode="shared", tx=tx, attribute_key="queue_static_exporter")
+
+        insert_rows("//tmp/q", [{"data": "sample"}] * 2)
+        self._flush_table("//tmp/q")
+
+        self._sleep_until_next_export_instant(export_period_seconds)
+        time.sleep(export_period_seconds)
+
+        # XXX(apachee): Is it really enough to check that retry is scheduled way before the next export iteration?
+        assert len(ls(export_dir)) == 0
+        abort_transaction(tx)
+        wait(lambda: len(ls(export_dir)) == 1)
+
+        self.remove_export_destination(export_dir)
+
+    @authors("apachee")
+    def test_export_segmentation(self):
+        _, queue_id = self._create_queue("//tmp/q")
+
+        export_dir = "//tmp/export"
+        self._create_export_destination(export_dir, queue_id)
+
+        dummy_strings = ["notepad", "vim", "far"]
+
+        for dummy_string in dummy_strings:
+            insert_rows("//tmp/q", [{"data": dummy_string}] * 2)
+            self._flush_table("//tmp/q")
+            # Sleep to make sure inserted rows are exported in different tables.
+            time.sleep(2)
+
+        set("//tmp/q/@static_export_config", {
+            "default": {
+                "export_directory": export_dir,
+                "export_period": 1 * 1000,
+            }
+        })
+
+        wait(lambda: len(ls(export_dir)) == 3)
+        self._check_export(export_dir, [[dummy_string] * 2 for dummy_string in dummy_strings])
+
+        self.remove_export_destination(export_dir)
+
+    @authors("apachee")
+    def test_late_flush(self):
+        _, queue_id = self._create_queue("//tmp/q")
+
+        export_dir = "//tmp/export"
+        self._create_export_destination(export_dir, queue_id)
+
+        export_period_seconds = 10
+        set("//tmp/q/@static_export_config", {
+            "default": {
+                "export_directory": export_dir,
+                "export_period": export_period_seconds * 1000,
+            }
+        })
+
+        self._sleep_until_next_export_instant(export_period_seconds)
+
+        insert_rows("//tmp/q", [{"data": "42"}] * 2)
+        self._flush_table("//tmp/q")
+        insert_rows("//tmp/q", [{"data": "13"}] * 2)
+
+        self._sleep_until_next_export_instant(export_period_seconds)
+
+        wait(lambda: len(ls(export_dir)) == 1)
+        self._check_export(export_dir, [["42"] * 2])
+
+        self._flush_table("//tmp/q")
+        wait(lambda: len(ls(export_dir)) == 2)
+        self._check_export(export_dir, [["42"] * 2, ["13"] * 2])
+
+        self.remove_export_destination(export_dir)
+
```

Comment:
More test ideas:
* Check behaviour of partial export tasks, where only some exports are successful, both to some natural causes (idk how to do it yet, maybe set some quota limits somehow? like set a separate account and a low node quota for it) and configuration limits (max exported tables per task). And also check multiple iterations here, so that tables are not re-exported for example. Also check alerts are propagated correctly here.
* Check ttl depending on how you implement it.
* Check throttler? It is a bit annoying to do, but I recently did it for master r/w user throttlers and I think it is a good practice to write tests for throttlers where possible. 

First one I am strongly requesting.